deployment_config/sandbox_vars.yml

```diff
@@ -0,0 +1,6 @@
+AUTH_CLIENT_ID: ${AUTH_CLIENT_ID}
```

Comment:
Dev and Sandbox have separate client id and secrets, so when you make the other changes please also have separate CircleCI variables for those two environments.

Reply:
Yes, there are separate CircleCI variables for dev and sandbox. When I update the variable passing I will update the naming of the sandbox secrets so that's clearer.